src/NSwag.SwaggerGeneration/Processors/Security/OperationSecurityScopeProcessor.cs

```diff
@@ -68,5 +73,14 @@ protected virtual IEnumerable<string> GetScopes(SwaggerOperationDescription oper
                 .SelectMany(a => ((string)a.Roles).Split(','))
                 .Distinct();
         }
+
+        /// <summary>Gets all customAttributes from a method.</summary>
+        /// <param name="methodInfo">The method information.</param>
+        /// <returns>The Attributes of the method.</returns>
+        private IEnumerable<Attribute> GetCustomAttributes(MethodInfo methodInfo)
+        {
+            return methodInfo.GetCustomAttributes().Concat(
```

Comment:
Maybe use GetCustomAttributes(true) (true => inherited attributes) instead of concat?

Reply:
I will try it, a busy day today. I'll come back to this.
I had copied this piece of code from your (line 58/59)